components/Token/Owners/Modal.tsx

```diff
@@ -58,10 +57,20 @@ const OwnersModal: FC<Props> = ({
       offset: (page - 1) * OwnerPaginationLimit,
     },
   })
-  const totalCount =
-    data?.ownerships?.totalCount || previousData?.ownerships?.totalCount
+
   // Reset pagination when the limit change or the modal visibility changes
   useEffect(() => setPage(1), [isOpen])
+
+  const hasNextPage = useMemo(
+    () => data?.ownerships?.pageInfo.hasNextPage,
+    [data?.ownerships?.pageInfo.hasNextPage],
+  )
+
+  const hasPreviousPage = useMemo(
+    () => data?.ownerships?.pageInfo.hasPreviousPage,
+    [data?.ownerships?.pageInfo.hasPreviousPage],
+  )
+
```

Comment:
No need for a memo for those variable, no calculation is made inside the memo.
```suggestion
  const hasNextPage = data?.ownerships?.pageInfo.hasNextPage
  const hasPreviousPage = data?.ownerships?.pageInfo.hasPreviousPage
```
You could also simply remove use of `hasNextPage` / `hasPreviousPage` by directly `data?.ownerships?.pageInfo.hasNextPage` / `data?.ownerships?.pageInfo.hasPreviousPage`, they are used only once.

Reply:
[remove not needed useMemo usages](https://github.com/liteflow-labs/starter-kit/pull/367/commits/7b667d97d135fbe297a9e2b0a67608eb4d4cda1d)